modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/context/DDMFormBuilderContextFactoryHelper.java

```diff
@@ -290,12 +290,16 @@ protected Value doCreateDDMFormFieldValue(
 	}
 
 	protected Value doCreateDDMFormFieldValue(
-		DDMFormFieldValidation ddmFormFieldValidation) {
+		DDMFormFieldValidation ddmFormFieldValidation, 
+		Set<Locale> availableLocales) {
 
 		JSONObject jsonObject = _jsonFactory.createJSONObject();
 
+		LocalizedValue errorMessage = ddmFormFieldValidation.getErrorMessage();
+
 		jsonObject.put(
-			"errorMessage", ddmFormFieldValidation.getErrorMessage()
+			"errorMessage", localizedValueToJSONObject(
```

Comment:
I removed the `localizedValueToJSONObject()` method since the logic is compact enough for it to be included in `doCreateDDMFormFieldValue()`.
`createOptions()` is a bit a bit different, as the options are stored as localized arrays:
`{"options" : {"en_US" : [{"label" : "value"}, {"label" : "value"} ...], "de_DE" : [{"label" : "value"}, {"label" : "value"} ...]}}
`
while my goal for errorMessage is to be structured like this:

`{ "errorMessage" : {"en_US" : "englisherror", "de_DE" : "germanerror", ...}}`